server/py/services/api/crud/pipelines.py

```diff
@@ -127,7 +131,17 @@ def list_pipelines(
             runs = [
                 mlrun_pipelines.models.PipelineRun(run) for run in response.runs or []
             ]
+
+            if isinstance(project, list) and project:
+                project_runs = []
+                for run in runs:
+                    run_project = self.resolve_project_from_pipeline(run)
+                    if run_project in project:
+                        project_runs.append(run)
+                runs = project_runs
```

Comment:
```suggestion
                runs = [run for run in runs if self.resolve_project_from_pipeline(run) in project]
```

Reply:
This code is being reverted anyway, pipelines sadly needs to be handled differently